tests/filesTest.js

```diff
@@ -205,6 +205,28 @@ describe('Main: Currently testing files management,', function () {
     })
   })
 
+  it('checking method: favorite filter', function (done) {
+    const newFile = testFolder + '/' + 'file.txt'
+
+    oc.files.putFileContents(newFile, testContent).then(status => {
+      expect(typeof status).toBe('object')
+      return oc.files.favorite(newFile)
+    }).then(status2 => {
```

Comment:
wondering if we should invest more in async-await in the future as I remember it could make this kind of chain more readable

Reply:
indeed worth a look :+1: